### PR TITLE
OCPBUGS-31263: Add missing PodSecurityViolation alert

### PR DESCRIPTION
**Pull request by Evan-Reilly**
These changes add a missing PodSecurityViolation alert found in OpenShift 4.15 (see https://github.ibm.com/alchemy-containers/armada-update/issues/5524 and https://www.redhat.com/en/blog/pod-security-admission-in-openshift-4.11 for more details).

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, use `fixes #<issue_number>(, fixes #<issue_number>, ...)` format, where issue_number might be a GitHub issue, or a Jira story*:
Fixes #

**Checklist**
- [x] Subject and description added to both, commit and PR.
- [x] Relevant issues have been referenced.
- [ ] This change includes docs. 
- [ ] This change includes unit tests.

### CODE REVIEW

**Review by rtheis — CHANGES_REQUESTED** [review]
@Evan-Reilly please see https://github.com/openshift/hypershift/pull/1406 for what else is needed to go along with this.

File: control-plane-operator/hostedclusterconfigoperator/controllers/resources/alerts/assets/podsecurity-violation.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Source: https://raw.githubusercontent.com/openshift/cluster-kube-apiserver-operator/release-4.15/bindata/assets/alerts/podsecurity-violation.yaml
```
Comment (rtheis) [inline control-plane-operator/hostedclusterconfigoperator/controllers/resources/alerts/assets/podsecurity-violation.yaml:1]: This source link is wrong.  In needs to be https://raw.githubusercontent.com/openshift/cluster-kube-apiserver-operator/release-4.15/bindata/assets/alerts/podsecurity-violations.yaml

**Review by rtheis — CHANGES_REQUESTED** [review]

**Review by rtheis — APPROVED** [review]
/lgtm

**Comment by bryan-cox** [comment]
/retest 
For Konflux

**Comment by bryan-cox** [comment]
This PR will also need a Jira ticket appended to the front of the title.

Would you also squash the commits please?

**Comment by bryan-cox** [comment]
/override "Red Hat Konflux"

Konflux is working again. The original error will not clear away without a rebase.

**Pull request merged**